mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -287,15 +408,26 @@ class DistributedLDAModel private (
     private[clustering] val globalTopicTotals: LDA.TopicCounts,
     val k: Int,
     val vocabSize: Int,
-    private[clustering] val docConcentration: Double,
-    private[clustering] val topicConcentration: Double,
+    protected[clustering] val docConcentration: Vector,
+    protected[clustering] val topicConcentration: Double,
+    protected[clustering] val gammaShape: Double,
     private[spark] val iterationTimes: Array[Double]) extends LDAModel {
 
   import LDA._
 
-  private[clustering] def this(state: EMLDAOptimizer, iterationTimes: Array[Double]) = {
-    this(state.graph, state.globalTopicTotals, state.k, state.vocabSize, state.docConcentration,
-      state.topicConcentration, iterationTimes)
+  private[clustering] def this(
+      state: EMLDAOptimizer,
+      iterationTimes: Array[Double],
+      gammaShape: Double) = {
+    this(
+      state.graph,
```

Comment:
style: No need to put 1 argument on each line like this (for method calls); we only do it for method & class headers.